src/lsdb/catalog/dataset/healpix_dataset.py

```diff
@@ -121,8 +121,11 @@ def _repr_data(self):
 
     @property
     def _repr_divisions(self):
-        name = f"npartitions={self._ddf.npartitions}"
-        divisions = pd.Index(self.get_ordered_healpix_pixels(), name=name)
+        pixels = self.get_ordered_healpix_pixels()
+        name = f"npartitions={len(pixels)}"
+        if len(pixels) == 0:
+            pixels = ["Empty Catalog"]
```

Comment:
```suggestion
        # Dask will raise an exception, preventing display,
        # if the index does not have at least one element.
        if len(pixels) == 0:
            pixels = ["Empty Catalog"]
```
I think Dask's expectation is odd enough to warrant a comment in this case.